package.json

```diff
@@ -38,11 +38,12 @@
     "json-stable-stringify": "1.0.1",
     "json3": "3.3.3",
     "mdn-browser-compat-data": "1.0.35",
+    "nodemon": "^2.0.4",
```

Comment:
Overall it shouldn't interfere with GAE, but just in case, I'll split this into `start-dev` instead.  Nodemon watches for any file changes and automatically reloads the server accordingly -- SUPER helpful for development purposes.